addons/binding/org.openhab.binding.heos/ESH-INF/thing/HeosSystem.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<!-- Thing Type -->
+	<bridge-type id="bridge">
+		<label>HEOS Bridge</label>
+		<description>The HEOS System Bridge</description>
+		<channels>
+			<channel typeId="reboot" id="Reboot"></channel>			
+			<channel typeId="buildGroup" id="BuildGroup"></channel>	
+			<channel typeId="rawCommand" id="RawCommand"></channel>
+		</channels>
+		<properties>
+			<property name="vendor">Denon</property>
+		</properties>
+		<config-description>
+			<parameter name="name" type="text">
+				<label>Player Type</label>
+				<description>Player Type of the Bridge.</description>
+				<required>false</required>
+			</parameter>
+			<parameter name="ipAddress" type="text">
+				<context>network_address</context>
+				<label>Network Address</label>
+				<description>Network address of the HEOS bridge.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="username" type="text">
+				<context>String</context>
```

Comment:
I thought I added this remark before, I hope you did not lose any/many commits? 

`context` `String` does not exist